diff --git a/crates/brush-ui/src/scene.rs b/crates/brush-ui/src/scene.rs
index f6439dc..43d0732 100644
--- a/crates/brush-ui/src/scene.rs
+++ b/crates/brush-ui/src/scene.rs
@@ -151,16 +151,7 @@ impl ScenePanel {
         splats: Option<Splats<MainBackend>>,
         interactive: bool,
     ) -> egui::Rect {
-        let mut size = ui.available_size();
-        let view = process.selected_view();
-        if let Some(view) = view {
-            let aspect_ratio = view.image.aspect_ratio();
-            if size.x / size.y > aspect_ratio {
-                size.x = size.y * aspect_ratio;
-            } else {
-                size.y = size.x / aspect_ratio;
-            }
-        }
+        let size = ui.available_size();
         let size = glam::uvec2(size.x.round() as u32, size.y.round() as u32);
 
         let (rect, response) = ui.allocate_exact_size(
